### PR TITLE
[WIP] Add processor info to configuration file

### PR DESCRIPTION
**Pull request by jgphpc**

### CODE REVIEW

File: config/cscs.py [hunk 1]
```diff
@@ -768,19 +768,17 @@
...
-                            'num_cpus_per_socket' 128,
-                            # +num_cores
-                            # +num_cores_per_socket
-                            # +num_numa_nodes
-                            # +num_cores_per_numa_node
-                            # 'num_sockets'
-                        },
-                    ],
+                    'processor': {
+                        'arch': 'AMD EPYC 7742 64-Core Processor',
```
Comment (ekouts) [inline config/cscs.py:772]: This should be compatible with [archspec](https://archspec.readthedocs.io/en/latest/index.html). After installing you can see the name with
```
python -c "import archspec.cpu; print(archspec.cpu.host())"
```

Comment (ekouts) [reply config/cscs.py:772]: I think it should be `zen2` in pilatus.
```suggestion
                        'arch': 'zen2',
```

**Comment by jgphpc** [comment]
ok, I will use this config file as a temporary solution.

**Comment by jgphpc** [comment]
see https://github.com/eth-cscs/reframe/pull/1991

**Pull request closed without merging**